hypertrace-graphql-gateway-service-metric-utils/src/main/java/org/hypertrace/graphql/utils/metrics/gateway/MetricAggregationMapConverter.java

```diff
@@ -39,7 +39,11 @@
   private Single<Entry<MetricLookupMapKey, MetricAggregation>> buildMetricAggregationEntry(
       MetricAggregationRequest aggregationRequest, Map<String, AggregatedMetricValue> resultMap) {
     return this.aggregationConverter
-        .convert(resultMap.get(aggregationRequest.alias()).getValue())
+        .convert(
+            resultMap
```

Comment:
The map should always have every entry (see our slack conversation from yesterday - an inactive entity should have an empty value only, not completely omit the request from the map)